### PR TITLE
added archived_order and proxy use

### PR DESCRIPTION
**Pull request by omid455**
if you edit line 171 and add proxies= after param, then you can connect with socks proxy.
Also added archived_orders functionality.

### CODE REVIEW

File: cexio/__init__.py [hunk 1]
```diff
@@ -168,6 +168,24 @@ def order_book(self, depth=1, market='BTC/USD'):
     def trade_history(self, since=1, market='BTC/USD'):
         return self.api_call('trade_history', None, market + '/?since=' + str(since))
 
-    def __post(self, url, param):
+    def __post(self, url, param): #this line changed added proxies, add after param to use socks procy (, proxies="socks5://175.155.68.86:38801")
```
Comment (fasetto) [inline cexio/__init__.py]: Don't understand why you didn't edit this line. 🤔 
I think you should add _proxies_ as optional paramater, like; 
```py
def __post(self, url, param, proxies={})
...
result = requests.post(url, data=param, headers={ 'User-agent': 'bot-cex.io-' + self.username }, proxies=proxies).json()
````
Something like this, maybe ?

**Review by fasetto — CHANGES_REQUESTED** [review]

File: cexio/__init__.py (hunk 1 above)
Comment (omid455) [reply cexio/__init__.py]: Sure, I am new to python and wasn't sure if that would work. just tested with proxies="xyz address" and it worked for me, I put that in the code as an option. You can go ahead and put that in the code.

Comment (fasetto) [reply cexio/__init__.py]: Maybe we shouldn't add proxy support yet. I don't think it is necessary tbh.

**Comment by fasetto** [comment]
Thanks for the PR btw.

**Comment by omid455** [comment]
> Thanks for the PR btw.

My pleasure, how long does it take for the new code to show up in PIP3 repository? I don' know how it works, does it happen automatically or you have to push it? thanks

**Comment by fasetto** [comment]
@omid455 I created this project long time ago. And didn't setup any CI/CD to automate the process.
I have to push it manually, but I can setup automated deploys via GitHub actions this week.

**Comment by fasetto** [comment]
It is done. Automated the process and updated the package on pypi.

**Comment by omid455** [comment]
> It is done. Automated the process and updated the package on pypi.

Well done, Thanks

**Pull request merged**